code/modules/paperwork/filingcabinet.dm

```diff
@@ -46,19 +46,19 @@
 	qdel(src)
 
 /obj/structure/filingcabinet/attackby(obj/item/P, mob/user, params)
-	if(P.w_class < WEIGHT_CLASS_NORMAL)
+	if(P.tool_behaviour == TOOL_WRENCH)

```

Comment:
Add an intent check here, like on tables that can only be unwrenched if on harm intent. That way you can still insert wrenches.